CHANGES_NEXT_RELEASE

```diff
@@ -1 +1 @@
-
+- ADD: add locale string number jexl function (#1737)
```

Comment:
Do we have some .md with the list of transformations?

In this case, that .md should be updated

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1740/commits/fe2e60f4353f7ecc9cbd65304208a014034cf33c